docs/adr/assets/ADR-003/examples/nodejs/main.ts

```diff
@@ -1,79 +1,58 @@
-import * as jwt from "jsonwebtoken";
-import axios from "axios";
+import { Octokit } from "octokit";
+import { Octokit as RestOctokit } from "@octokit/rest";
+import { createAppAuth } from "@octokit/auth-app";
 import * as fs from "fs";
 
-interface Installation {
-  id: number;
-  account: {
-    login: string;
-  };
-}
+export const getOctokit = async (
+  appId: string,
+  privateKey: string,
+  orgName: string
+): Promise<RestOctokit> => {
+  const appOctokit = new Octokit({
+    authStrategy: createAppAuth,
+    auth: {
+      appId,
+      privateKey,
+    },
+  });
+  const installations = await appOctokit.request("GET /app/installations");
+  for (const d of installations.data) {
+    //@ts-ignore
+    if (d.account.login === orgName) {
+      const installationId = d.id;
+      const installationOctokit = new RestOctokit({
+        authStrategy: createAppAuth,
+        auth: {
+          appId,
+          privateKey,
+          installationId,
+        },
+      });
+      return installationOctokit;
```

Comment:
Just out of curiosity why do we need to use the octokit from the @octokit/rest library?
Do you know why we can't use the App from the regular octokit library?
Does the RestOctokit have the same features ast he one returned from the method
`app.getInstallationOctokit()`

Reply:
I believe the @octokit/rest was a commented-out leftover from our session last week that I didn't remove but instead extended. Now, it's gone. Thanks @amaanibn-nasar1-nhs